### PR TITLE
issue2991: add FactStore getter

### PR DESCRIPTION
**Pull request by bedaka**

### CODE REVIEW

File: factcast-core/src/main/java/org/factcast/core/ReadFactCast.java [hunk 1]
```diff
@@ -66,4 +67,11 @@ Optional<Fact> fetchByIdAndVersion(@NonNull UUID id, int versionExpected)
   // @Deprecated(since = "0.5.6", forRemoval = true)
   @Deprecated
   ReadFactCast retry(int maxAttempts, long minimumWaitIntervalMillis);
+
+  /**
+   * Internal API: subject to change - use at your own risk
+   *
+   * @since 0.7.8
+   */
+  FactStore getFactStore();
```
Comment (uweschaefer) [inline factcast-core/src/main/java/org/factcast/core/ReadFactCast.java:76]: we're not using getXyz() for getters within factcast, but xyz()

File: factcast-core/src/main/java/org/factcast/core/ReadFactCast.java [hunk 2]
```diff
@@ -66,4 +67,12 @@ Optional<Fact> fetchByIdAndVersion(@NonNull UUID id, int versionExpected)
   // @Deprecated(since = "0.5.6", forRemoval = true)
   @Deprecated
   ReadFactCast retry(int maxAttempts, long minimumWaitIntervalMillis);
+
+  /**
+   * Internal API: subject to change - use at your own risk
+   *
+   * @since 0.7.8
```
Comment (otbe) [inline factcast-core/src/main/java/org/factcast/core/ReadFactCast.java:74]: 0.7.10?

Comment (uweschaefer) [reply factcast-core/src/main/java/org/factcast/core/ReadFactCast.java:74]: right

Comment (uweschaefer) [reply factcast-core/src/main/java/org/factcast/core/ReadFactCast.java:74]: will commit

**Review by otbe — APPROVED** [review]
LGTM

**Comment by uweschaefer** [comment]
closes #2991

**Pull request merged**